pkg/query/querier_test.go

```diff
@@ -843,7 +843,6 @@ func (s *mockedSeriesIterator) At() (t int64, v float64) {
 	return sample.t, sample.v
 }
 
-// TODO(rabenhorst): Needs to be implemented for native histogram support.
 func (s *mockedSeriesIterator) AtHistogram() (int64, *histogram.Histogram) {
```

Comment:
I see we still panic in this method but removed the comment. Do we still need to implement anything?

Reply:
Will restore it. Rn `mockedSeriesIterator` does not support histograms and if we need it it needs to be implemented.